src/pybind/mgr/volumes/fs/operations/volume.py

```diff
@@ -227,6 +247,18 @@ def list_volumes(mgr):
         result.append({'name': f['mdsmap']['fs_name']})
     return result
 
+def info(path):
```

Comment:
`info` is essentially getting number of pending subvolume deletions. Doesn't the name too generic for that ?